documentation/modules/security/con-securing-client-authentication.adoc

```diff
@@ -82,6 +76,31 @@ data:
   user.password: # Password for protecting the PKCS #12 archive file
 ----
 
+When you configure a client, you specify the following:
+
+* *Truststore* properties for the public cluster CA certificate to authenticate with the Kafka cluster
+* *Keystore* properties for the user authentication credentials to verify the client
+
+The configuration depends on the file format (PEM or PKCS #12).
+This example uses the PKCS #12 archive files, and the passwords required to access the credentials. 
```

Comment:
You call it `archive` on multiple places, but I do not think it is right. It is `store` or just `file`. I think calling it `archive` suggests it is some kind of ZIP for example, but it is more than that.

Reply:
Agree, let's use `store` it's more appropriate.